crates/epaint/src/tessellator.rs

```diff
@@ -864,19 +873,29 @@ fn stroke_path(
     feathering: f32,
     path: &[PathPoint],
     path_type: PathType,
-    stroke: Stroke,
+    stroke: &PathStroke,
     out: &mut Mesh,
 ) {
     let n = path.len() as u32;
 
-    if stroke.width <= 0.0 || stroke.color == Color32::TRANSPARENT || n < 2 {
+    if stroke.width <= 0.0 || stroke.color == ColorMode::TRANSPARENT || n < 2 {
         return;
     }
 
     let idx = out.vertices.len() as u32;
 
+    // expand the bounding box to include the thickness of the path
+    // this tiny bit of margin is here because a rect doesn't think it contains a point if a pos' component is on a boundary. why? i have no idea.
+    let bbox = Rect::from_points(&path.iter().map(|p| p.pos).collect::<Vec<Pos2>>())
+        .expand((stroke.width / 2.0) + feathering + 1.0);
```

Comment:
Probably because of rounding errors. If you change the `.contains(x)` test to `distance_to_pos(x) < 0.1` it will probably work. That is better than adding an arbitrary `+ 1.0` here methinks

Reply:
When doing some debugging prints, the edge case happens when they're dead equal, which `.contains()` doesn't seem to handle. I'll try the distance check and see if that helps at all, though.

Reply:
`Rect::contains` should handle it:

```
    pub fn contains(&self, p: Pos2) -> bool {
        self.min.x <= p.x && p.x <= self.max.x && self.min.y <= p.y && p.y <= self.max.y
    }
```

Reply:
assertion failure is spitting out the following, really strange: `passed rect [[-49.0 -49.0] - [59.0 59.0]] didn't contain point [59.1 4.3]`, so it's exactly 0.1 out. I can just have it be <= instead of just <, but that doesn't feel right.

Reply:
So, it seems that the error gets larger as the path width increases, I have no idea what's going on, but the problem seems to be cropping up starting at line 971.